pkg/oc/bootstrap/bindata.go

```diff
@@ -11486,7 +11511,8 @@ var _examplesQuickstartsRailsPostgresqlPersistentJson = []byte(`{
   },
   "message": "The following service(s) have been created in your project: ${NAME}, ${DATABASE_SERVICE_NAME}.\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/openshift/rails-ex/blob/master/README.md.",
   "labels": {
-    "template": "rails-pgsql-persistent"
+    "template": "rails-pgsql-persistent",
+    "app": "rails-pgsql-persistent"
```

Comment:
The `app` label should be parameterized so that when templates are used in conjunction to form a single application we can make the `app` labels match.